core/unit_test/TestTeam.hpp

```diff
@@ -1529,6 +1529,38 @@ struct TestScratchAlignment {
 
 }  // namespace
 
+namespace {
+
+template <class ExecSpace>
+struct TestTeamPolicyHandleByValue {
+  using scalar     = double;
+  using exec_space = ExecSpace;
+  using mem_space  = typename ExecSpace::memory_space;
+
+  TestTeamPolicyHandleByValue() { test(); }
+
+  void test() {
+#if defined(KOKKOS_ENABLE_CXX11_DISPATCH_LAMBDA)
+    const int M = 1, N = 1;
+    Kokkos::View<scalar **, mem_space> a("a", M, N);
+    Kokkos::View<scalar **, mem_space> b("b", M, N);
+    Kokkos::deep_copy(a, 0.0);
+    Kokkos::deep_copy(b, 1.0);
+    Kokkos::parallel_for(
+        "test_tphandle_by_value",
```

Comment:
Typo?  I would not bother with the label anyway.

Reply:
@ndellingwood Can you take care of this?

Reply:
@dalg24 @masterleinad I force pushed the update to remove the label and open/close of anonymous namespace; rebased to a single commit

Reply:
hm, my local rebased changes said they successfully pushed to this branch but not showing up here in github

Reply:
@masterleinad I looked at this branch on your fork and the SHA there matches the changes I made locally and force pushed, but not showing up in the PR, any suggestions to help get this PR to update? I'm not sure what I messed up...
https://github.com/masterleinad/kokkos/commit/13d020d773674a0e50b12c954b82604c53b90a46